osu.Game/Overlays/NowPlayingOverlay.cs

```diff
@@ -278,12 +294,34 @@ protected override void Update()
                 progressBar.EndTime = track.Length;
                 progressBar.CurrentTime = track.CurrentTime;
 
+                // Process the Current Minute and Second
+                CurrentMinutes = Math.Floor((track.CurrentTime / 1000) / 60).ToString();
+                CurrentSeconds = Math.Floor((track.CurrentTime / 1000) % 60).ToString();
```

Comment:
2nd pair of brackets are redundant ( the ones around the " track.currentime / 1000")